test/vcr/vcr_cassettes/service_bus/test_0001_should_be_able_to_send_a_message_with_SAS_to_a_queue.yml

```diff
@@ -0,0 +1,169 @@
+---
```

Comment:
the key is not sent, it's used to sign a string and produce a token, going in the headers "Authorization", which is deleted from the recording.